nuxeo-runtime/nuxeo-stream/src/main/java/org/nuxeo/lib/stream/computation/FilteringPolicy.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * (C) Copyright 2019 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     pierre
+ */
+
+package org.nuxeo.lib.stream.computation;
+
+/**
+ * @since 11.1
+ */
+public interface FilteringPolicy {
+
+    default boolean shouldSkip(String key, ComputationContext context) {
```

Comment:
We shouldn't have `default` on this method as it's the only contract to implement for this interface.

Reply:
Would you prefer a concrete class with default implementation ?